carbon-design-system/carbon-website
# docs: update figma kit page with card icons

**Changed**
- Update the Figma kit page to include Figma icons on each resource card.

---

**Testing**
- Go to `Designing` --> `Design kits` --> `Figma` tab
  - Make sure resource card icons appear correctly.
